src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -22,7 +21,7 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
-        FindCommand findCommand = null;
```

Comment:
Perhaps we could leave this as null but do a assert statement somewhere down the line?